recipes-multimedia/gstreamer/gstreamer1.0-plugins-base/0001-meson-viv-fb-code-must-link-against-libg2d.patch

```diff
@@ -0,0 +1,44 @@
+From 8153e86dac04076fb80fc272c8374d4ba019fa3c Mon Sep 17 00:00:00 2001
+From: Zan Dobersek <zdobersek@igalia.com>
+Date: Wed, 4 Nov 2020 14:02:10 +0100
+Subject: [PATCH] meson: viv-fb code must link against libg2d
+
+Find the libg2d library and link against it, if necessary, avoiding
+linking errors for couple of libg2d symbols.
+
+Upstream-Status: Pending
```

Comment:
Please also put a link to your MR here, that way it would be visible when it gets merged in.

Reply:
This change is against the NXP fork of gst-plugins-base. I don't know of any public issue trackers for that repository, so I mailed the maintainers for further information.

Reply:
> This change is against the NXP fork of gst-plugins-base.

So this is rather a change in NXP-specific part of `gstreamer1.0-plugins-base`? In this case the `Upstream-Status:` should be set to `Inappropriate [NXP-specific]`.

> I don't know of any public issue trackers for that repository, so I mailed the maintainers for further information.

Have you posted it to some list or directly to maintainer? If it is the later, then maybe Cc'ing _meta-freescale@yoctoproject.org_ would also be helpful to track the status of that patch.

Reply:
I'll use `Inappropriate [i.MX specific]`, as already used in a few patch files.